src/containers/ProvisionerForm.js

```diff
@@ -8,6 +8,8 @@ import GithubService from '../components/provisionerForm/GithubService';
 import ProjectName from '../components/provisionerForm/ProjectName';
```

Comment:
is it OK to use relative paths like this in JS? ... What if my CWD is somewhere else? ...